src/System.Drawing.Primitives/src/System/Drawing/Point.cs

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Globalization;
+
+namespace System.Drawing
+{
+    /// <devdoc>
```

Comment:
Can you change these to be normal XML comments (i.e. `<summary>`, etc.)? VS doesn't understand this kind as far as I can tell.